packages/synapse-react-client/src/components/dataaccess/SubmissionPage/SubmissionPage.test.tsx

```diff
@@ -334,4 +395,210 @@ describe('Submission Page tests', () => {
     // When an ACL exists, don't show the ACT as the reviewer
     expect(screen.queryByText(mockActTeam.name)).not.toBeInTheDocument()
   })
+
+  describe('Viewing page as an accessor', () => {
+    it('supports canceling a SUBMITTED request', async () => {
+      mockGetUserAccessApproval.mockReturnValue(
+        getUseQueryErrorMock(
+          new SynapseClientError(
+            404,
+            'Not found',
+            expect.getState().currentTestName!,
+          ),
+        ),
+      )
+
+      renderComponent({
+        submissionId: SUBMITTED_SUBMISSION_ID,
+        isReviewer: false,
+      })
+
+      await screen.findByText('Status')
+      await screen.findByText('SUBMITTED')
+
+      const cancelRequestButton = await screen.findByRole('button', {
+        name: 'Cancel Request',
+      })
+
+      await userEvent.click(cancelRequestButton)
+
+      expect(mockCancelSubmissionModal).toHaveBeenLastCalledWith(
+        {
+          open: true,
+          submissionId: String(SUBMITTED_SUBMISSION_ID),
+          accessRequirementId: String(mockManagedACTAccessRequirement.id),
+          onClose: expect.any(Function),
+        },
+        expect.anything(),
+      )
+      expect(mockAccessRequirementList).not.toHaveBeenCalled()
+    })
+
+    it('supports updating an APPROVED and unexpired request', async () => {
+      const futureDate = new Date()
+      futureDate.setFullYear(futureDate.getFullYear() + 1)
+      mockGetUserAccessApproval.mockReturnValue(
+        getUseQuerySuccessMock({
+          ...mockAccessApproval,
+          expiredOn: futureDate.toISOString(),
+          state: 'APPROVED',
+        }),
+      )
+      renderComponent({
+        submissionId: APPROVED_SUBMISSION_ID,
+        isReviewer: false,
+      })
+
+      await screen.findByText('Status')
+      await screen.findByText('APPROVED')
+
+      const updateRequestButton = await screen.findByRole('button', {
+        name: 'Update Request',
+      })
+
+      await userEvent.click(updateRequestButton)
```

Comment:
Like to verify anything after clicking the update button?

Reply:
Yes, I did in the other tests but missed this one. Will add in a follow-up